### ISSUE TITLE
SVGs do not include the width/height values

### ISSUE THREAD

**Issue by ddbrown30** [game-icons/icons#992]
SVG icons created by the site do not have the width and height fields set. They are set in the style field but not on their own. This can cause issues in certain browsers and with certain apps. Anything I make here I have to manually edit to include those fields.

Can you please update the SVG generation to add those fields?

**Comment by aruguy** [comment]
Fixing this would be a big win. Placing any SVG icons generated by the site onto the game canvas in FoundryVTT causes the scene to crash because of the missing dimensions.

**Comment by ddbrown30** [comment]
Yeah, my problem case is also with Foundry. I have to manually fix every icon before I can use it.
